module/applications/actor/character-sheet-2.mjs

```diff
@@ -82,14 +83,16 @@ export default class ActorSheet5eCharacter2 extends ActorSheetV2Mixin(ActorSheet
     const isUpdate = (context === "update") || (context === "updateActor");
     const hp = foundry.utils.getProperty(data ?? {}, "system.attributes.hp.value");
     if ( isUpdate && (hp === 0) ) this._toggleDeathTray(true);
+    const [bastion] = this.element.find('nav.tabs [data-tab="bastion"]');
+    if ( bastion ) bastion.toggleAttribute("hidden", this.object.system.details.level < 5);
```

Comment:
Doesn't seem to actually be hiding the tab.

Reply:
Not able to reproduce this

Reply:
Are you running V13? Because it hides properly in V13 but is visible in V12:

<img width="772" alt="Screenshot 2024-11-01 at 08 36 42" src="https://github.com/user-attachments/assets/c1540eac-faee-496b-b806-d00f3d7dab67">